src/clusterfuzz/_internal/fuzzing/fuzzer_selection.py

```diff
@@ -135,8 +135,23 @@ def get_fuzz_task_payload(platform=None):
   if not mappings:
     return None, None
 
+  logs.log(f'Mappings: {mappings}')
+  # The environment variable contains the list of jobs seperated by comma
```

Comment:
nit: `The environment variable containing a list of comma-separated jobs.`

Reply:
Done